<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ORGANIFILL</class>
 <widget class="QWidget" name="ORGANIFILL">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>539</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="barra_menu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>201</width>
     <height>511</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>700</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background:grey</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="btn_nueva_forma">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Forma</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_guardar">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Guardar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_exportar">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Exportar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_volver">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Volver</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_informes">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Informes</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
